python-sdk/docs/CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## 1.2.3
+
+### Bug fixes
+* Fix the dataframe operator behavior of appending to it's output table instead of replacing. When `if_exists` is set to `replace`, replace the table rather than append [#1260](https://github.com/astronomer/astro-sdk/issues/1260)
+* Previously, `PostgresHook` did not support `database` kwarg. After schema is renamed to database in airflow as per this [PR](https://github.com/apache/airflow/pull/26744), now the user needs to set database to the table's database. [#1276](https://github.com/astronomer/astro-sdk/pull/1276)
+
+### Improvement:
+* Use `DefaultExtractor` from OpenLineage. `OPENLINEAGE_EXTRACTORS` is not required to be set by user to use OpenLineage. [#1223](https://github.com/astronomer/astro-sdk/issues/1223)
```

Comment:
can you think from a user's perspective, and rephrase this sentence?

Reply:
I also think this should be part of 1.3 instead of 1.2.3

Reply:
Sure. We can update documentation accordingly as well for this: https://github.com/astronomer/astro-sdk/issues/1292

Reply:
Added this to 1.3.0 milestone